onnxruntime/test/onnx/TestCase.cc

```diff
@@ -1035,6 +1035,10 @@ std::unique_ptr<std::set<BrokenTest>> GetBrokenTests(const std::string& provider
   // std::set<std::string> broken_tests_keyword_set = {};
 
   if (provider_name == "cuda") {
+#ifdef ENABLE_TRAINING_CORE
```

Comment:
optional: If this is specific to a particular cuda HW, shall we use that as a check instead of `ENABLE_TRAINING_CORE`?

Reply:
We are not sure whether it is limited to some hardware or cudnn versions, need more investigation.